inst/rstudio/templates/project/collapse_duplicates_project_files/index.Rmd

```diff
@@ -120,6 +127,11 @@ input_size <- nrow(db)
 
 ```
 
+# Mask sequence alignments from 5' end to a specific IMGT position
```

Comment:
```suggestion
```{r eval=params$imgt_position!=0, results='asis'}
cat("# Mask sequence alignments from 5' end to a specific IMGT position\n")
```

```

Reply:
This section will only be created if the user provides the right parameter